microsoft-edge/index.yml

```diff
@@ -238,36 +238,35 @@ conceptualContent:
               url: ./extensions-chromium/getting-started/part1-simple-extension.md
               itemType: how-to-guide
         # Card
-        - title: Progressive Web Apps
+        - title: Progressive Web Apps (PWAs)
```

Comment:
which .md files in pwa card in hub page?

Reply:
hub page > pwa card: 
* Deleted "Debug a pwa".
* Moved "Services workers" & "web app manifest" to Get Started section of card.
* Added "publish to store", in How-to section of card.